docs/volumes.adoc

```diff
@@ -88,3 +90,103 @@ Ports for the Storage Unit are assigned automatically (`49252` to `49452`) while
 Volume vol1 created successfully
 ID: 02b6a3a4-e704-47ee-bb05-e541a561a921
 ----
+
+=== Volume Expansion and Rebalance
+
+Expand a already created Kadalu Storage Volume with Storage units from multiple Storage nodes.
+
+Example,
+
+Create a pure distributed Kadalu Storage Volume `vol1` with 3 different Storage nodes and 3 Storage units.
+
+[source,console]
+----
+[root@server1 ~]# kadalu volume create DEV/vol1 \
+           server1.example.com:/exports/vol1/s1 \
+           server2.example.com:/exports/vol1/s2 \
+           server3.example.com:/exports/vol1/s3
+Volume vol1 created successfully
+ID: 03c6a3a4-e704-47ee-bb05-e541a561a875
+----
+
+Expand Kadalu Storage Volume `vol1` by adding more Storage units.
+
+- Note:
+
+  Volume type, distribute group count, replica count, data and redundant storage unit count should be the same as when the Kadalu Storage Volume was created.
+
+  For example above Kadalu Storage Volume being of type pure distribute, it has 3 distribute group. Hence during volume expansion Kadalu Storage expects distribute group being a multiple of 3.
+
+
+[source,console]
+----
+[root@server1 ~]# kadalu volume expand DEV/vol1 \
+         server1.example.com:/exports/vol1/s1_e \
+         server2.example.com:/exports/vol1/s2_e \
+         server3.example.com:/exports/vol1/s3_e
+Volume vol1 expanded successfully
+ID: 03c6a3a4-e704-47ee-bb05-e541a561a875
+
+Proceed to the rebalancing of volume vol1 by following the below steps.
+To start the rebalancing of volume: kadalu volume rebalance-start DEV/vol1.
+To force stop the rebalancing of volume: kadalu volume rebalance-stop DEV/vol1.
+----
+
+==== Rebalance
+
+During expansion of volume, Kadalu Storage also fixes the layout of all expanded storage units, which is to help and keep the storage units ready for migration of data.
```

Comment:
Fix layout is done mainly to allow creating the new files in the new storage units.

Reply:
> which is to help and keep the storage units ready for migration of data.

Yeah, I meant here that new storage units would have dirs creation process would be started after expansion so it would be ready for migration of data.
Will add that context here for clearer understanding.

Reply:
Updated